src/plugins/irc-events/link.js

```diff
@@ -166,18 +166,15 @@ function fetch(uri, cb) {
 			}
 		})
 		.on("error", function() {})
-		.pipe(es.map(function(data, next) {
+		.on("data", (data) => {
 			length += data.length;
 			if (length > limit) {
 				req.response.req.abort();
+			} else {
+				result = Buffer.concat([result, data]);
 			}
-			next(null, data);
-		}))
-		.pipe(es.wait(function(err, data) {
-			if (err) {
-				return cb(null);
-			}
```

Comment:
`end`'s callback has no `err` argument, so I just removed this. Maybe we need to do this in `.on("error", function() {})` above instead?